src/llv8.cc

```diff
@@ -1268,6 +1268,14 @@ StackTrace::Iterator StackTrace::end() {
 
 StackTrace::StackTrace(JSArray frame_array, Error& err)
     : frame_array_(frame_array) {
+  if (!frame_array.Check()) {
+    Error::PrintInDebugMode(
+        "JS Array is not a valid object");
```

Comment:
Do you think this message would be more helpful if it mentioned that the invalid JS array is an error stack?

Reply:
I thought about that, but I want to change our PrintInDebugMode to also print the function where it was called, so I decided to keep like this until I make that change.